src/WP_CLI/SearchReplacer.php

```diff
@@ -83,7 +83,7 @@ private function run_recursively( $data, $serialised, $recursion_level = 0, $vis
 				}
 			}
 
-			$unserialized = is_string( $data ) ? @unserialize( $data ) : false;
+			$unserialized = is_serialized( $data ) ? @unserialize( $data ) : false;
```

Comment:
This generally seems fine. Could we get a test case for it?

Also, let's internalize `is_serialized()` to a private method on this class. As far as I can tell, we don't have any other WordPress-specific dependencies in this class, so better to keep things that way for now.

Reply:
Yes, during a `search-replace`, WP might potentially be broken, so it's best to copy the function over.